client/src/components/Dialogs/TestEditProfile.js

```diff
@@ -36,15 +36,17 @@ export const useStyles = makeStyles((theme) => ({
 const EditProfile = ({ create, profile }) => {
```

Comment:
I'm a little confused why this filename was updated to TestEditProfile

Reply:
The team wants to handle edit profile in another way. but I still needed this component to set up a chef profile so that I can have a recipe to work with. That's why I need to have it but it will probably be removed from the final product.